sdcm/mgmt/cli.py

```diff
@@ -436,17 +436,8 @@ def create_backup_task(self, dc_list=None,  # pylint: disable=too-many-arguments
         if legacy_args:
             cmd += f" {legacy_args}"
         res = self.sctool.run(cmd=cmd, parse_table_res=False)
-        if not res.stdout:
```

Comment:
i think you could keep this check too, as it could be an issue with `sctool` and it could report `res.ok` but no output from the command... the next one (`res.stderr`) should be replaced with the new one (`res.ok`)

Reply:
I think it is reasonable enough to count on res.ok . There could be hundreds of issues with std.out so no point to over check. in any case the test will probably fail immediately in one of the following steps.

Reply:
i don't think you are right, because we count on this stdout to create a `BackupTask` as in the following code:
`task_id = res.stdout.strip()`

if the stdout will be empty for any reason (even if `res.ok` is True), the `BackupTask` will fail, because it needs the `task_id`

Reply:
Exactly as i just wrote above: "the test will probably fail immediately in one of the following steps"
(1) i estimate the chances of that to happen to 1/10000000.
(2) there will be no flakiness or unclear issue following that since it will be very clear exception in the right same place.